src/virtualenv/seed/embed/pip_invoke.py

```diff
@@ -12,12 +15,15 @@ def __init__(self, options):
     def run(self, creator):
         cmd = self.get_pip_install_cmd(creator.exe, creator.interpreter.version_release_str)
         env = pip_wheel_env_run(creator.interpreter.version_release_str)
+        logging.debug("pip seed by running: %s", Cmd(cmd, env))
         process = Popen(cmd, env=env)
         process.communicate()
+        if process.returncode != 0:
+            raise RuntimeError("failed seed")
 
     def get_pip_install_cmd(self, exe, version):
-        cmd = [str(exe), "-m", "pip", "install", "--only-binary", ":all:"]
-        for folder in {get_bundled_wheel(p, version).parent for p in self.packages}:
+        cmd = [str(exe), "-m", "pip", "-q", "install", "--only-binary", ":all:"]
+        for folder in {get_bundled_wheel_non_zipped(p, version).parent for p in self.packages}:
             cmd.extend(["--find-links", str(folder)])
```

Comment:
unrelated / not a regression in this patch -- the ordering here is non-deterministic

Reply:
hmm 👍 will need to see